cli/command/engine/check.go

```diff
@@ -50,54 +51,47 @@ func newCheckForUpdatesCommand(dockerCli command.Cli) *cobra.Command {
 }
 
 func runCheck(dockerCli command.Cli, options checkOptions) error {
-	ctx := context.Background()
-	client, err := dockerCli.NewContainerizedEngineClient(options.sockPath)
-	if err != nil {
-		return errors.Wrap(err, "unable to access local containerd")
+	if !isRoot() {
+		return errors.New("must be privileged to activate engine")
```

Comment:
Shouldn't the message be something like "must be privileged to check the engine" ?